builders/ethereum/dev-env/hardhat.md

```diff
@@ -97,9 +97,9 @@ For this example, you can leave the Solidity compiler version to `0.8.28`; howev
 
 Next, you'll need to modify your configuration file to add the network configurations for the network you want to deploy your contract to. For Moonbeam networks, you'll need to specify the following:
 
-- `url` - the [RPC endpoint](/builders/get-started/endpoints/){target=\_blank} of the node
-- `chainId` - the chain ID, which is used to validate the network
-- `accounts` - the accounts that can be used to deploy and interact with contracts. You can either enter an array of the private keys for your accounts or use an [HD Wallet](https://github.com/ethereumbook/ethereumbook/blob/develop/05wallets.asciidoc#hierarchical-deterministic-wallets-bip-32bip-44){target=\_blank}
+- `url`: The [RPC endpoint](/builders/get-started/endpoints/){target=\_blank} of the node.
+- `chainId`: The chain ID, which is used to validate the network.
+- `accounts`: The accounts that can be used to deploy and interact with contracts. You can either enter an array of the private keys for your accounts or use an [HD Wallet](https://github.com/ethereumbook/ethereumbook/blob/develop/src/chapter_5.md#hierarchical-deterministic-wallets-bip-32bip-44){target=\_blank}.
```

Comment:
If you're fixing these things, then we should also put the list item titles in bold please!